modules/govuk/manifests/node/s_docker_management.pp

```diff
@@ -0,0 +1,34 @@
+# == Class: govuk::node::s_docker_management
+#
+class govuk::node::s_docker_management inherits govuk::node::s_base {
+
+  include ::docker
+
+  $etcd_image = 'quay.io/coreos/etcd'
+  $etcd_image_version = 'v3.0.13'
+
+  ::docker::image { $etcd_image:
+    ensure    => 'present',
+    image_tag => $etcd_image_version,
+    require   => Class['docker'],
+  }
+
+  ::docker::run { 'etcd':
+    image   => "${etcd_image}:${etcd_image_version}",
+    ports   => ['2379:2379','2380:2380'],
```

Comment:
Maybe `2379` and `2380` should be extracted to variables. They're magic numbers and duplicated multiple times in this class definition.